injectproxy/routes.go

```diff
@@ -88,7 +89,7 @@ func WithErrorOnReplace() Option {
 
 // WithLabelValue enforces a specific value for the multi-tenancy label.
 // If not specified, the value has to be provided as a URL parameter.
-func WithLabelValue(value string) Option {
+func WithLabelValue(value []string) Option {
```

Comment:
The comment would also need an update.
```suggestion
func WithLabelValues(value []string) Option {
```